pom.xml

```diff
@@ -116,6 +116,7 @@
         <module>presto-testing-server-launcher</module>
         <module>presto-plugin-toolkit</module>
         <module>presto-resource-group-managers</module>
+        <module>presto-sac-plugin</module>
```

Comment:
what about presto-access-control?
presto-sac-plugin doesn't really describe what it is